bin/setup_venv.sh

```diff
@@ -195,12 +194,15 @@ else
 fi
 
 # Install general package requirements
-pip3 install --requirement "${script_dir}/../pulumi/python/requirements.txt"
+PIPENV_VERBOSITY=-1 PIPENV_PIPFILE="${script_dir}/../pulumi/python/Pipfile" pipenv install
+
 # Install local common utilities module
-pip3 install "${script_dir}/../pulumi/python/utility/kic-pulumi-utils" &&
-  rm -rf "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/.eggs" \
-    "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/build" \
-    "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/kic_pulumi_utils.egg-info"
+pip3 install "${script_dir}/../pulumi/python/utility/kic-pulumi-utils"
+
+rm -rf "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/.eggs" \
+  "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/build" \
+  "${script_dir}/../pulumi/python/utility/kic-pulumi-utils/kic_pulumi_utils.egg-info"
+
```

Comment:
**annotation:**
Removing the `&&` prevents an issue where if the pip install does not work the script would not fail.  The `rm` operation should not fail either way.